jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/DeploymentScanner.java

```diff
@@ -273,7 +273,11 @@ void addScannerListener(Scanner.Listener listener)
     /**
      * Configure the Environment specific Deploy settings.
      *
-     * @param name the name of the environment.
+     * @param name the name of the environment. This is usually one of "core", "static",
```

Comment:
These names only exist if you are using jetty standalone.
They come into existence due to the configuration you are using.
The names themselves actually exist in the `${jetty.home}/modules/*.mod` files.

On Jetty Embedded (an super common use case) the environment names do not exist.
So documenting them as existing in the javadoc isn't correct.